crates/ide_db/src/tests/sourcegen_lints.rs

```diff
@@ -176,16 +217,37 @@ fn generate_descriptor_clippy(buf: &mut String, path: &Path) {
         push_lint_completion(buf, &lint_ident, &doc);
     }
     buf.push_str("];\n");
+
+    buf.push_str(r#"pub const CLIPPY_LINT_GROUPS: &[LintGroup] = &[];"#);
```

Comment:
:)

Reply:
I want to fix this before merge

Reply:
What exactly is the problem here?

Reply:
clippy has groups, so I've added those now